docs/pages/enroll-resources/machine-id/deployment/kubernetes.mdx

```diff
@@ -124,6 +124,11 @@ $ kubectl apply -f ./k8s-rbac.yaml
 
 ## Step 3/5. Create a join token
 
+<Admonition type="note">
```

Comment:
I'm usually not a big fan of starting a section with an Admonition, since Admonitions are comments for edge cases, and there's often an expectation when beginning a section that the text will explain what a section is about. Is there another way to organize this information?

Reply:
I think this is one of the rare edge cases, it could be confusing imo to add this elsewhere. A resolution is planned in the future, so I'd like to not not make this extremely prominent if we can prevent it (we could also add multiple workarounds for example)